README.md

```diff
@@ -135,8 +139,12 @@ If you use Mitiq in your research, please reference the [Mitiq preprint][arxiv]
 }
 ```
 
+A list of papers citing Mitiq can be found [here][papers_with_mitiq].
+
 [arxiv]: https://arxiv.org/abs/2009.04417
 
+[papers_with_mitiq]: https://mitiq.readthedocs.io/en/stable/research.html#papers-citing-or-using-mitiq
```

Comment:
Now this link doesn't work but I guess the idea is that it will work after we do the next release, right?

Reply:
yes.